login/link/keycloak_idp.go

```diff
@@ -16,38 +16,38 @@ import (
 {"identityProvider":"rhd","userId":"<USER_ID>","userName":"<USERNAME>"}
 */
 
-// KeycloakLinkIDPRequest represents standard Keycloak User profile api request payload
-type KeycloakLinkIDPRequest struct {
+// OAuthServiceLinkIDPRequest represents standard OAuth service User profile api request payload
+type OAuthServiceLinkIDPRequest struct {
```

Comment:
keycloak_idp.go is for Keycloak specific code. It works only with Keycloak. I would not rename it.